### DISCUSSION TITLE
How to change data-tip text color on tooltip.?

### DISCUSSION THREAD

**Discussion by crescent1** [saadeghi/daisyui#945] [Q&A] [answered]
how to change data-tip text color,..? 

`<div class="tooltip tooltip-info text-base-100" data-tip="Detail">
      <button class="btn btn-square btn-info btn-sm text-base-100">info</button>
    </div>`

trying add `text-base-100` but data-tip text color still black

**Comment by rhscjohn-dev** [comment]
The Daisyui class: tooltip-info generates the following css style`.tooltip-info {
  --tooltip-color: hsla(var(--in));
  --tooltip-text-color: hsl(var(--inc));
}`

I would suggest that you set the  Daisyui  color name  ` 'info-content` to the color that you want. See [Daisyui Colors](https://daisyui.com/docs/colors/) in the docs.

**Comment by saadeghi** [comment, marked as answer]
You can use CSS variables as Tailwind CSS class names like this:
```css
[--tooltip-text-color:#ff0]
```
Example: https://play.tailwindcss.com/Tr4Xte02ES
